warehouse/sponsors/models.py

```diff
@@ -0,0 +1,39 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from sqlalchemy import Boolean, Column, String, sql
+from sqlalchemy.types import PickleType
+from sqlalchemy_utils.types.url import URLType
+
+from warehouse import db
+from warehouse.utils.attrs import make_repr
+
+
+class Sponsor(db.Model):
+    __tablename__ = "sponsors"
+    __repr__ = make_repr("name")
+
+    name = Column(String, nullable=False)
+    service = Column(String)
+    # activity should be a list of strings
+    activity = Column(PickleType, nullable=False)
```

Comment:
I see the note above, but we definitely do not want to be using Pickles <3

Reply:
postgres has an array type or you can use jsonb, or obviously a related table that has it's defaults set to join on load.

Reply:
Oh sure! I didn't realize we were using postgres! I'll change this by an array field then :+1